pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -109,6 +125,34 @@ func (i *Controller) processNextObject(objWorkqueue workqueue.RateLimitingInterf
 			return fmt.Errorf("error while splitting meta namespace key %s: %v", key, err)
 		}
 
+		runtimeObj, _ := objGetter(ns, name)
+		switch runtimeObj.(type) {
+		case *k8sv1.Service:
+			service := runtimeObj.(*k8sv1.Service)
+			switch i.processServiceStatus(service) {
+			case Ignore:
```

Comment:
We can return without annotating globalIp to this service. In other words, we're adding a new condition for services that won't get globalIP.